src/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- <link rel="stylesheet" href="index.css"> -->
+    <title>Leaderboard</title>
+</head>
+
+<body>
+    <main>
+        <header>
+            <h1>Leaderboard</h1>
+        </header>
+        <div class="board">
+        <div class="score">
+            <div class="score-board-header"><h2>Recent Scores</h2><input type="button" id="RefreshBtn" value="Refresh"></div>
+            <div id="scores"></div>
+        </div>
+        <div class="score-form">
+            <div class="score-form-header"><h2>Add Your Score</h2></div>
+            <form action="post" id="score-form">
+               <label for="names"> <input type="text" id="names" placeholder="Your Name">
+               <label for="scoressTXT"> <input type="text" id="scoresTXT" placeholder="Your Scores"><br>
+                <input type="button" class="" id="btnAdd" value="Sumit">
+            </form>
+        </div>
+        </div>
+    </main>
+
+<script src="main.js"></script>
+    <!-- <script src="index.js" type="module"></script> -->
+    </body>
```

Comment:
- [ ] [OPTIONAL] Please make sure to remove any commented code which makes your code more professional.